content/devs.json

```diff
@@ -85,7 +85,7 @@
     "contacts": [
       {
         "type": "website",
-        "url": "https://codekcv.now.sh/"
+        "url": "http://codekcv.github.io/"
```

Comment:
Why? cc @codekcv

Reply:
I think this part of the JSON is not mine 😅

I think i accidentally touched and made minor changes on it

Reply:
Hello, please revert this change. It should be `https://codekcv.now.sh/`. Thanks, @jromest for noticing this.